_notes/rft.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Request For Teachers
+---
+# Request For Teacher (RFT) Template
+
+By: [who?]
+
+Students currently [Current Context]. 
+However, [Identified Gap]. 
+Ideally they should be able to [Ideal Outcome].
+
+Optional Details:
+[- Possible implementation ideas (e.g thoughts you have on how to achieve the ideal outcome)
+- Assessment criteria (e.g any tangible reflections you expect to see)
+- Engagement format (e.g looking for workshop/internship/lecture)
+- Engagement timeline (e.g available dates)
+- Etc]
+
+---
+SAMPLE RFT 1
+By: [Mr Jo]
+
+Students currently [are trying to launch their own startup ideas]. 
+However, [their copywriting skills are lacking, resulting in poor marketing outcomes]. 
+Ideally they should be able to [write compelling copy to showcase their ideas].
+
+Optional Details:
+[A 2h intro workshop during the September holidays would be great! My students should walk away being able to write better product titles.]
+
+---
+SAMPLE RFT 2
+By: [Mr Tan]
+
+Students currently [have 3d printers in the homeroom.]
+However, [they lack enough practice around CAD tooling to fully utilise the printers.] 
+Ideally, they should be able to [proficiently make and modify everyday items for printing after.]
+
+Optional Details:
+[Nil]
+
+---
+SAMPLE RFT 3
+By: [Ms Lim]
+
+Students currently [have to study "good governance" for social studies]. 
+However, [they find it really dry and just try to memorise the content, leading to poor understanding and application]. 
+Ideally they should be able to [learn about governance in a more engaging way].
+
+Optional Details:
+[It would be great if students get to learn more about other "Bad" government policies to contrast what "Good" governance mean.]
+
+------------------------------------------------------------------------------------
+# Request for Learners (RFL) Template
+
+I want to help students [long-term outcome] through [creative method].
+
+Optional Details:
+- This is in response to which RFT?
+- Please list your key lesson objectives
+- What is your exact lesson plan?
+- What is your exact activity plan?
+- What are some measurable metrics you would like to track?
```

Comment:
i feel for measure it has to tie in with feedback 

how do you wish for students to respond with various time bounds
in a week 
in a month 
by the end of the term etc